src/Packetery/Module/Options/TransientPurger.php

```diff
@@ -45,11 +44,11 @@ public function purge(): void {
 			$sites = Plugin::getSites();
 			foreach ( $sites as $site ) {
 				switch_to_blog( $site );
-				$this->optionsRepository->deleteExpiredTransientsByPrefix( Checkout::TRANSIENT_CHECKOUT_DATA_PREFIX );
+				wc_delete_expired_transients();
```

Comment:
Opravdu si můžeme dovolit mazat transienty WP a jiných pluginů?

Reply:
mazání všech expirovaných transientů bez ohledu na to z jakého jsou pluginu bych opravdu nedělal.  můžeme smazat něco, co eshop z nějakého důvodu potřebuje a co nám nepatří.  WC je také smaže až když to někdo chce a klikne na tlačítko.  Smazal bych pouze naše transienty.  Ať už jsou expirované nebo nejsou